blog/blog_gsoc_2023.md

```diff
@@ -0,0 +1,109 @@
+(gsoc_2023_presentations)=
+# Introducing the 2023 PyMC Google Summer of Code Students
+

```

Comment:
The post directive is missing, so it is not being listed in the blog section for the PR preview build.

Preview: https://pymcio--90.org.readthedocs.build/blog.html

Reference to post directive: https://github.com/pymc-devs/pymc.io/blob/main/blog/blog_gsoc_2022.md?plain=1#L4-L12. Not 100% sure about the metadata, @reshamas should probably weigh in on this.